src/components/Error/style.css

```diff
@@ -0,0 +1,32 @@
+.error {
```

Comment:
Consider, since this can have impact on developers customising the css:
you can make your selectors simpler if you like, considering that they will already be namespaced with `Error` automatically.

Something like  `.container, .message, .instruction, .icon`